cvat/apps/engine/media_extractors.py

```diff
@@ -510,3 +553,190 @@ def _is_zip(path):
         'unique': True,
     }
 }
+
+
+class ValidateDimension:
+
+    def __init__(self, path=None):
+        self.dimension = "2d"
+        self.path = path
+        self.related_files = {}
+        self.image_files = {}
+        self.converted_files = []
+
+    @staticmethod
+    def validate_pcd(path):
+        """
+            Validate the point cloud file along with version check
+            On failure it is considered as PhotoCD format
+        """
+
+        pcd_version = ["0.7", "0.6", "0.5", "0.4", "0.3", "0.2", "0.1", ".7", ".6", ".5", ".4", ".3", ".2", ".1"]
```

Comment:
It should be a feature of a module which parses PCD files. I don't like the idea to spread the code over the file.

Reply:
We have included all the parsing of PCD/BIN files in the class ValidateDimension.